src/wkb_writer.rs

```diff
@@ -7,7 +7,7 @@ use geos_sys::*;
 use std::convert::TryFrom;
 use std::sync::Arc;
 
-/// The `WKBWriter` type is used to generate `HEX` or `WKB` formatted output from [`Geometry`].
+/// The `WKBWriter` type is used to generate `HEX` or `WKB` formatted output from [`Geometry`](crate::Geometry).
```

Comment:
Why this change?

Reply:
All these doc string changes are because the links are broken (Geometry is not actually imported in that module), e.g. without the addition when running cargo doc:

> warning: unresolved link to `Geometry`
>   --> src\wkb_writer.rs:10:85
>    |
> 10 | /// The `WKBWriter` type is used to generate `HEX` or `WKB` formatted output from [`Geometry`]
>    |                                                                                     ^^^^^^^^ no item named `Geometry` in scope
>    |
>    = note: `#[warn(rustdoc::broken_intra_doc_links)]` on by default
>    = help: to escape `[` and `]` characters, add '\' before them like `\[` or `\]`

Reply:
I see, that's weird it wasn't seen before. Thanks for the fix!